### PR TITLE
Added retranslate scripts

### PR DESCRIPTION
**Pull request by mutaphysis**
Take a look at an example replacement in https://github.com/sozialhelden/wheelmap-react-frontend/pull/94/commits/a1274dbfdb40a89ab3d82fc56990fc04545432a7

### CODE REVIEW

File: retranslate/.babelrc [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+  "plugins": [
+    "syntax-class-properties",
+    "syntax-object-rest-spread",
+    "syntax-jsx",
+    "syntax-flow",
+    ["./retranslate/babel-plugin-retranslate/dist-plugin.js", {
+      "replacement": "../public/i18n/de.txt"
```
Comment (opyh) [inline retranslate/.babelrc]: I guess this should be `en_US.txt`?

File: retranslate/rewrite-pos/rewrite-pos.js [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+      } else {
+        console.warn(`new source translation missing for '${newValue.msgid}'`);
+        rewrittenContext[key] = newValue;
+      }
+
+      // retranslate plurals
+      if (newValue.msgid_plural) {
+        if (rewriteValue.msgstr[1]) {
+          const newKey = rewriteValue.msgstr[1];
+          newValue.msgid_plural = newKey;
```
Comment (opyh) [inline retranslate/rewrite-pos/rewrite-pos.js]: Does this work with languages that have more than two plural forms, e.g. Russian?

File: retranslate/run.sh [hunk 3]
```diff
@@ -0,0 +1,37 @@
+# setup environment
+export NEW_SOURCE_PO=${NEW_SOURCE_PO:-'./public/i18n/en_US.txt'}
+export NODE_ENV=development
+
+# ensure i18n & src folder are reset 
+git checkout -- public/i18n/ src
```
Comment (opyh) [inline retranslate/run.sh]: I'd be afraid that this script accidentally my changes.

image

How about if this exits if there are changes and lets the user do the cleanup themselves?

Maybe like this?

```bash
test -z "$(git status --porcelain)" || (echo "Your repo must be clean"; exit 1)
```

File: retranslate/rewrite-pos/rewrite-pos.js (hunk 2 above)
Comment (mutaphysis) [reply retranslate/rewrite-pos/rewrite-pos.js]: Only the first plural form is used as the new plural id, this should not matter for the other plural forms

File: retranslate/.babelrc (hunk 1 above)
Comment (mutaphysis) [reply retranslate/.babelrc]: Will fix, and add script to package json

File: retranslate/run.sh (hunk 3 above)
Comment (mutaphysis) [reply retranslate/run.sh]: Oh yes, this was more for dev, will add

**Pull request merged**